cookbook/agent_concepts/user_control_flows/agentic_user_input.py

```diff
@@ -0,0 +1,110 @@
+"""🤝 Human-in-the-Loop: Allowing users to provide input externally
+
+This example shows how to use the UserControlFlowTools to allow the agent to get user input dynamically.
```

Comment:
Let's clarify it's used to get user input **to clarify how to use a tool** ?

Reply:
Maybe `UserInputTools` or smt is more clear?

Reply:
I'm thinking to add more tools to this toolkit in future, so I want to keep it broad. I'll extend the docstring though